tests/php/Models/LinkTest.php

```diff
@@ -329,4 +336,71 @@ public function linkUrlCasesDataProvider(): array
             ],
         ];
     }
+
+    function linkDefaultTitleDataProvider(): array
+    {
+        return [
+            'page link' => [
+                'page-link-1',
+                SiteTreeLink::class,
+                'PageLink1'
```

Comment:
```suggestion
                'identifier' => 'page-link-1',
                'class' => SiteTreeLink::class,
                'expected' => 'PageLink1'
```
This makes it way clearer what each of these values represents. Please do this to all scenarios in this method.

Reply:
Updated